pkg/protocol/engine/ledger/ledger.go

```diff
@@ -28,6 +28,7 @@ type Ledger interface {
 	AddUnspentOutput(unspentOutput *utxoledger.Output) error
 
 	ConflictDAG() conflictdag.ConflictDAG[iotago.TransactionID, iotago.OutputID, BlockVoteRank]
+	MemPool() mempool.MemPool[BlockVoteRank]
```

Comment:
why is this exposed now?

Reply:
We need to obtain the MemPool from the dashboard and tipselection to be able to load transaction metadata.